toothpick-runtime/src/main/java/toothpick/Toothpick.java

```diff
@@ -28,6 +28,19 @@ private Toothpick() {
     throw new RuntimeException("Constructor can't be invoked even via reflection.");
   }
 
+  /**
+   * Find scope by this {@code name} in current scopes tree.
+   * @return true if scope is exist.
+   */
+  public static boolean hasScope(Object name) {
+    if (name == null) {
+      throw new IllegalArgumentException("null scope names are not allowed.");
+    }
+
+    Scope scope = MAP_KEY_TO_SCOPE.get(name);
+    return scope != null;
```

Comment:
why not `MAP_KEY_TO_SCOPE.contains(name)`? It seems like `get` would be needed in cases where it is possible to have a `name` mapped to a `null` scope object. Is this possible in toothpick? (I don't know internals enough to answer that)

Reply:
Hmm. May be you're right. But scope can't be null for Toothpick.